dev-libs/libaddressinput/libaddressinput-0~git.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="Google’s postal address library that powering Android and Chromium"
+DESCRIPTION="libaddressinput can provide information about \
+what input fields are required for a correct address input form for \
+any country in the world and can validate an address to highlight \
+input errors like missing required fields or invalid values."
+HOMEPAGE="https://github.com/googlei18n/libaddressinput"
+COPYRIGHT="2014 Google Inc.
+	2015 THL A29 Limited, a Tencent company
+	2015 Milo Yip
+	2006-2013 Alexander Chemeris
+	2002 JSON.org"
+LICENSE="Apache v2
+	JSON
+	MIT"
+REVISION="1"
+srcGitRev="a575c63de328d0c72237c55d1a05e191c245122a"
+SOURCE_URI="https://github.com/googlei18n/libaddressinput/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f92b38a0291372aa97306626f2941485bd1c00bd3cc7e242a1cb9fcd50b62212"
+SOURCE_FILENAME="libaddressinput-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="libaddressinput-$srcGitRev"
+googletestVersion="1.8.0"
+SOURCE_URI_2="https://github.com/google/googletest/archive/release-$googletestVersion.tar.gz"
+CHECKSUM_SHA256_2="58a6f4277ca2bc8565222b3bbd58a177609e9c488e8a72649359ba51450db7d8"
+SOURCE_FILENAME_2="gtest-$googletestVersion.tar.gz"
+SOURCE_URI_3="https://chromium.googlesource.com/chromium/src/tools/grit/+archive/b033674.tar.gz"
+CHECKSUM_SHA256_3="2e9ef06cb237426be60b4031dff82f414bd9a5ded775c2af1fbac51a7fa345a9"
+SOURCE_FILENAME_3="grit.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libaddressinput$secondaryArchSuffix = $portVersion
+	lib:libaddressinput$secondaryArchSuffix = portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libre2
+	"
+
+PROVIDES_devel="
+	libaddressinput${secondaryArchSuffix}_devel
+	devel:libaddressinput$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	libaddressinput$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:rapidjson == 1.1.0
```

Comment:
Please don't requires a specific version like this. Packages are subjected to updates, and doing so would causes trouble for maintainers once they decide to drop that particular version

Reply:
I think it should be <= 1.1.0. libaddressinput cannot be build with the newest mainstream source code form github. rapidjson-1.1.0 is the latest release. But I didn't tested with the earlier release of rapidjson.